CHANGES.md

```diff
@@ -178,6 +178,8 @@ To be released.
  -  `BencodexFormatter` became able to serialize `BigInteger`.  [[#159]]
  -  Made `Swarm` possible to configure its network `appProtocolVersion` and,
     to ignore peers if their version is different.  [[#167]], [[#170]]
+ -  Added `IActionContext.Miner` property and renamed
+    `Block<T>.RewardBeneficiary` to `Block<T>.Miner`[[#173]], [[#174]]
```

Comment:
Could you separate this into two bullets?  To me two changes seem independent each other.

Reply:
Also this statement lacks a period in the end.

Reply:
I fixed change log and amended commit.